tests/stubtest_third_party.py

```diff
@@ -16,9 +16,14 @@
 from utils import colored, get_mypy_req, make_venv, print_error, print_success_msg
 
 
-def run_stubtest(dist: Path, *, verbose: bool = False, specified_platforms_only: bool = False) -> bool:
+def run_stubtest(
+    dist: Path, *, parser: argparse.ArgumentParser, verbose: bool = False, specified_platforms_only: bool = False
+) -> bool:
     dist_name = dist.name
-    metadata = read_metadata(dist_name)
+    try:
+        metadata = read_metadata(dist_name)
+    except ValueError as e:
```

Comment:
Maybe it's better to use a custom exception class here. `ValueError` is a pretty common exception, so there is a risk that an error gets swallowed, especially when we expand `read_metadata()` at some point.

Reply:
Yeah, I wondered about this. I'll add a `NoSuchStubError` class.